### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by danil-maksimenko**
- [DEMO LINK](https://danil-maksimenko.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://danil-maksimenko.github.io/layout_search-bar-airbnb/report/html_report/)

- [x] Icon implemented using background-image CSS property
- [x] Inputs are written inside of 'form' tag with correctly passed attributes
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,109 @@
...
+
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: var(--main-text-colot);
+}
+
+input {
```
Comment (Esceype) [inline src/style.css:24]: Set it in search-form__input

File: src/style.css [hunk 2]
```diff
@@ -1 +1,109 @@
...
+  font-weight: 800;
+}
+
+:root {
+  --main-text-colot: #3D4E61;
+
+}
+
+html {
+  font-family: Avenir, sans-serif;
```
Comment (Esceype) [inline src/style.css:19]: Add fallback font

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,35 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label">
+        <span class="search-form__image search-form__image--big"></span>
```
Comment (Esceype) [inline src/index.html:20]: Just set all styles in this selectors to search-form__input, you don`t need this additional span here and in the small form

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ!
Read the following comments

File: src/style.css (hunk 1 above)
Comment (danil-maksimenko) [reply src/style.css:24]: I need to set color: var(--main-text-colot) in search-form__input ?

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,35 @@
...
+        <input
+          type="text"
+          class="search-form__input search-form__input--big"
```
Comment (VitaliyBondarenko1982) [inline src/index.html:23]:
```suggestion
          class="search-form__input"
```
you don't need --big modifier.
set all styles in .search-form__input selector and change necessary for --small modifier.
same for other elements

File: src/style.css (hunk 1 above)
Comment (VitaliyBondarenko1982) [reply src/style.css:24]: do not style by tags. move all styles for input to .search-form__input selector.
box-sizing better to set in * selector

File: src/style.css [hunk 5]
```diff
@@ -1 +1,109 @@
...
+  font-weight: 800;
+}
+
+.search-form__input::placeholder {
+  color: var(--main-text-color);
+}
+
+.search-form__input--big {
+  padding-left: 62px;
+  font-size: 16px;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:103]: move this styles in .search-form__input selector

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Please check comments.
And demo_links - they are not works.
Try to deploy again.
If will have some problems with deploy - feel free to ask in chat.

File: src/style.css [hunk 6]
```diff
@@ -1 +1,94 @@
...
+.search-form__image--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-form__input {
+  outline: none;
+  box-sizing: border-box;
```
Comment (VitaliyBondarenko1982) [inline src/style.css:70]: box-sizing better to set for * selector.

```
* {
  box-sizing: border-box
}
```

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Well done.

**Pull request closed without merging**